book/book.toml

```diff
@@ -0,0 +1,28 @@
+[book]
+authors = ["Josh Rotenberg"]
```

Comment:
Is the authors field necessary? If so, please change this to 
```suggestion
authors = ["The Rust Clippy Developers"]
```

If not, just remove it.

Reply:
Worth having in there I think.